repos/fdbt-site/src/pages/products/exports.tsx

```diff
@@ -234,11 +236,37 @@ export const getServerSideProps = async (ctx: NextPageContextWithSession): Promi
     if (exportStarted) {
         updateSessionAttribute(ctx.req, SELECT_EXPORTS_ATTRIBUTE, undefined);
     }
+    const exportNames = await getS3Exports(noc);
```

Comment:
If we are no longer using the code above this (the SELECT_EXPORTS_ATTRIBUTE etc) then we need to remove it, from here and also from the API where it is set.